doc/Language/io-guide.pod6

```diff
@@ -68,49 +71,51 @@ can handle them, so there's no need to convert them.
 
 Let's make some files and write and read data from them! The
 L«C<spurt>|/routine/spurt» and L«C<slurp>|/routine/slurp» routines write and
-read the data in one chunk. Unless you're working with very large files that
-are difficult to store entirely in memory all at the same time, these two
-routines are for you.
+read the data in one chunk respectively. Unless you're working with very large
+files that are difficult to store entirely in memory all at the same time,
+these two routines are for you.
 
 =for code
 "my-file.txt".IO.spurt: "I ♥ Perl!";
 
 The code above creates a file named C<my-file.txt> in the current directory
 and then writes text C<I ♥ Perl!> into it. If Perl 6 is your first language,
-celebrate your accomplishment! Try to open the file you created with some other
-program to verify what you wrote with your program. If you already know
+celebrate your accomplishment! Try to open the file you created with some
```

Comment:
*a* text editor?

Reply:
Just to be sure, do you mean "some text editor" -> "a text editor"?

Reply:
Yep.